### PR TITLE
Tooltips

### LINKED ISSUE #394: Add Build Table Tooltips
screen shot 2016-10-31 at 1 36 30 pm

### PR DESCRIPTION
**Pull request by MattHJensen**
Closes #394. 

@brittainhard, could you review?

### CODE REVIEW

**Comment by brittainhard** [comment]
Sorry the the delay, @MattHJensen, taking a look now.

File: webapp/apps/constants.py [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
+DIAGNOSTIC_TOOLTIP = "Key variables in the computation of tax liabilities."
+DIFFERENCE_TOOLTIP = "Key variables that highlight the differences between two tax plans."
+PAYROLL_TOOLTIP = " Include employee and employer payroll taxes in output."
+INCOME_TOOLTIP = "Include individual income taxes in output."
+BASE_TOOLTIP = "Current law as defined by default parameters."
+REFORM_TOOLTIP = "The reform proposal as defined by user-provided parameters."
+EXPANDED_TOOLTIP = "The tab variable includes adjusted gross income, non-taxable interest income, taxable investment income excluded from AGI, non-taxable social security benefits, the employer’s share of FICA, and above the line adjustments to AGI on form 1040."
+ADJUSTED_TOOLTIP = "Not yet available."
+INCOME_BINS_TOOLTIP = "Tax units are binned according to the income group to which they belong."
+INCOME_DECILES_TOOLTIP = " Tax units are binned according to the income decile to which they belong."
```
Comment (brittainhard) [inline webapp/apps/constants.py]: This line has the non-ascii quotation character (unicode?) ’ which is causing problems. Changing that back to the ascii ' will fix this.

File: webapp/apps/constants.py [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
-ADJUSTED_TOOLTIP = "Adjusted income tooltip"
-INCOME_BINS_TOOLTIP = "Income bins tooltip"
-INCOME_DECILES_TOOLTIP = "Income deciles tooltip"
+DIAGNOSTIC_TOOLTIP = "Key variables in the computation of tax liabilities."
+DIFFERENCE_TOOLTIP = "Key variables that highlight the differences between two tax plans."
+PAYROLL_TOOLTIP = " Include employee and employer payroll taxes in output."
+INCOME_TOOLTIP = "Include individual income taxes in output."
+BASE_TOOLTIP = "Current law as defined by default parameters."
+REFORM_TOOLTIP = "The reform proposal as defined by user-provided parameters."
+EXPANDED_TOOLTIP = "The tab variable includes adjusted gross income, non-taxable interest income, taxable investment income excluded from AGI, non-taxable social security benefits, the employer’s share of FICA, and above the line adjustments to AGI on form 1040."
```
Comment (brittainhard) [inline webapp/apps/constants.py]: This line has the non-ascii quotation character (unicode?) ’ which is causing problems. Changing that back to the ascii ' will fix this.

**Comment by MattHJensen** [comment]
Thanks @brittainhard, updated.

**Pull request merged**